test/e2e/http/client.test.ts

```diff
@@ -24,9 +24,9 @@ describe('http/client', () => {
   // we put a timeout of 10s since it takes around 4s for Dapr to boot up
   beforeAll(async () => {
     client = new DaprClient(daprHost, daprPort, CommunicationProtocolEnum.HTTP, {
-      isKeepAlive: false
+      isKeepAlive: false,
+      daprApiToken: 'token',
```

Comment:
```suggestion
      daprApiToken: 'securedtoken',
```

Note, for the tests to work you also need to edit package.json and add `DAPR_API_TOKEN` environment variable, similar to `TEST_SECRET_1`